<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>97</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>329</width>
    <height>97</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>329</width>
    <height>97</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Change value</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icons/CoPUI.ico</normaloff>../Icons/CoPUI.ico</iconset>
  </property>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="keyL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>20</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="valueL">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>141</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
